src/main/java/io/axoniq/axonserver/connector/event/impl/BufferedAggregateEventStream.java

```diff
@@ -31,12 +34,15 @@
         implements AggregateEventStream {
 
     private static final Event TERMINAL_MESSAGE = Event.newBuilder().setAggregateSequenceNumber(-1729).build();
+    private static final int TIMEOUT_MILLIS = Integer.parseInt(System.getProperty("AGGREGATE_TAKE_EVENT_TIMEOUT_MILLIS",
```

Comment:
Nit: I'd rename this constant to `TAKE_TIMEOUT_MILLIS`. Just `TIMEOUT_MILLIS` is a bit to generic for me.